bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/WWZJets/WWZTo4L_4f_NLO/WWZTo4L_4f_NLO_proc_card.dat

```diff
@@ -5,5 +5,5 @@ define lep- = e- mu- ta-
 # exclude s-channel top contribution to avoid
 # overlap with TTZ sample
 generate p p > w w lep+ lep- $$ t t~ h [QCD] 
```

Comment:
What is $$ t t~ , How can it get such contributions

Reply:
I assume, that's what original requester intended. To exclude `t t~ h` s-channel.

> How can it get such contributions

@Saptaparna do you know about this?